app/src/main/res/layout/receive_tab_preboarding.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think `receive_tabs_preboarding` should be in the nodpi folder (though I don't entirely understand it). From [SO](https://stackoverflow.com/questions/34156957/what-is-the-difference-between-anydpi-and-nodpi), nodpi is:

> These are density-independent resources. The system does not scale resources tagged with this qualifier, regardless of the current screen's density.

Which means that the asset will appear have a different density-independent size at different density levels thus appearing smaller or larger on screen at different density levels. What we want to happen is that they fill the same amount of space of on the screen regardless of the density level.

Our other assets are in xhdpi and it seems to be working fine: maybe we should just put this in there.